lib/rack/deflater.rb

```diff
@@ -99,7 +100,7 @@ def each(&block)
         gzip = ::Zlib::GzipWriter.new(self)
         gzip.mtime = @mtime if @mtime
         # @body.each is equivalent to @body.gets (slow)
-        if @body.is_a? ::File
```

Comment:
We might have to revisit how this works because it seems like it won't handle streaming bodies correctly, and in any case we probably don't want to handle streaming bodies with the implementation here.

Reply:
Agreed.